python/SockJSAdapter.py

```diff
@@ -209,12 +211,21 @@ def run():
         os.environ["RSTUDIO_PANDOC"] = input["pandocPath"]
 
     sys.path.insert(0, input["appDir"])
-    app_module = importlib.import_module("app")
-    app = getattr(app_module, "app")
+
+    app_file = os.path.join(input["appDir"], "app.py")
+    if is_express_app(app_file, None):
+        print("Shiny Express detected")
```

Comment:
Just curious: is this `print` statement and the one below intentional, or leftover debugging info?

Reply:
I had intended for it to be intentional. But stdout is actually thrown away by Shiny Server, so right now it's useless. I should remove it or have it print to stderr. (Probably remove it, I guess?)

Reply:
Also debating whether to have stdout redirect to stderr (and thus the error log) in Shiny for Python apps. There's reasons why we don't do this for R, but I think it is more defensible in Python, especially given that some of the framework's log messages are currently on stdout 😬